symphony-bdk-examples/bdk-app-spring-boot-example/src/main/resources/application.yaml

```diff
@@ -0,0 +1,21 @@
+
+bdk:
+  host: devx1.symphony.com
```

Comment:
Maybe try not to put internal server address here (could be localhost or a more generic/public pod maybe?)